### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by JevanChandra**
Please help review my code 
@AdriantoHartono
@a-rosa
@Riston-gitting-gud
@MikeLeongZhiFai
Thanks.

### CODE REVIEW

File: cp1404practicals/prac_06/car.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_06/car.py:1]: The class code has been written nicely👌

File: cp1404practicals/prac_06/guitar.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_06/guitar.py:1]: Good class code, the code has been written neatly 👍

File: cp1404practicals/prac_06/guitars.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_06/guitars.py:1]: Good code, it can show the output as expected 👌

File: cp1404practicals/prac_06/languages.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_06/languages.py:1]: Good code and simple, nice 👍

File: cp1404practicals/prac_06/programming_language.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_06/programming_language.py:1]: Good class code, it already written nicely👌

File: cp1404practicals/prac_06/used_cars.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_06/used_cars.py:1]: Good, the code can give an output as expected 👍

File: cp1404practicals/prac_06/guitar.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        return f"{self.name} ({self.year}) : ${self.cost}"
+
+    def get_age(self):
+        return 2022 - self.year
+
+    def is_vintage(self):
+        if self.get_age() >= 50:
+            return True
+        else:
+            return False
```
Comment (a-rosa) [inline cp1404practicals/prac_06/guitar.py:17]: Missing docstring on all function

File: cp1404practicals/prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    print("\n ... snip ... \n")
+    print("These are my guitars: ")
+    for i, guitar in enumerate(guitars, 1):
+        if guitar.is_vintage():
+            vintage_string = "(vintage)"
+        else:
+            vintage_string = ""
+        print(f"Guitar {i}: {guitar.name:>{length}} {guitar.year}, worth ${guitar.cost:10,.2f} {vintage_string}")
+
+
```
Comment (a-rosa) [inline cp1404practicals/prac_06/guitars.py:28]: Forgotten docstring on the function

File: cp1404practicals/prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+def get_longest_name(guitars):
+    length = 0
+    for guitar in guitars:
+        if len(guitar.name) > length:
+            length = len(guitar.name)
+    return length
+
+
+main()
```
Comment (a-rosa) [inline cp1404practicals/prac_06/guitars.py:37]: I dont think you are supposed to have a function to find the longest name in the problem, but the code is correct

File: cp1404practicals/prac_06/programming_language.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        self.year = year
+
+    def is_dynamic(self):
+        if self.typing == "Dynamic":
+            return True
+        else:
+            return False
+
+    def __str__(self):
+        return f"{self.name}, {self.typing} Typing, Reflection={self.reflection}, First appeared in {self.year}"
```
Comment (a-rosa) [inline cp1404practicals/prac_06/programming_language.py:15]: Missing docstring on all function

File: cp1404practicals/prac_06/car.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_06/car.py:1]: Simple code, easy to read as well!

File: cp1404practicals/prac_06/guitar.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_06/guitar.py:1]: Easy to read and understand!

File: cp1404practicals/prac_06/guitars.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_06/guitars.py:1]: Code working well!

File: cp1404practicals/prac_06/languages.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_06/languages.py:1]: Easy to understand!

File: cp1404practicals/prac_06/programming_language.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_06/programming_language.py:1]: Class is well written!

File: cp1404practicals/prac_06/used_cars.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_06/used_cars.py:1]: Code is working fine!

**Pull request merged**